vcell-util/src/main/java/org/vcell/util/Extent.java

```diff
@@ -102,6 +102,13 @@ public double getZ() {
 	return z;
 }
 
+public void rescaleInPlace(double scaleFactor, int dimension){
+	switch (dimension){
+		case 3: this.x *= scaleFactor; this.y *= scaleFactor; this.z *= scaleFactor; break;
+		case 2: this.x *= scaleFactor; this.y *= scaleFactor; break;
+		case 1: this.x *= scaleFactor; break;
+	}
+}

```

Comment:
this method is no longer needed - and so a bit confusing if left in. (same comment as Origin)